tpp/src/http/AuthHttpClient.js

```diff
@@ -401,6 +401,50 @@ class AuthHttpClient extends CoreAuthHttpClient {
         return this._instance(request);
     }
 
+    /**
+     * Get url to bank authorization page for a token request.
+     *
+     * @param bankId {string} Bank Id
+     * @param tokenRequestId {string} Token request id
+     * @returns {Object} response to the api call
+     */
+    async getBankAuthUrl(bankId, tokenRequestId){
+        const request = {
+            method: 'post',
+            url: `/banks/${bankId}/token-requests/${tokenRequestId}`,
+        };
+        return this._instance(request);
+    }
+
+    /**
+     * Forward the callback from the bank (after user authentication) to Token.
+     *
+     * @param bankId {string} bank Id
+     * @param query {string} HTTP query string
+     * @returns {Object} response to the api call
+     */
+    async onBankAuthCallback(bankId, query){
+        const request = {
+            method: 'post',
+            url: `/banks/${bankId}/callback?${query}`,
```

Comment:
should be `query=${query}`?

Reply:
yes I actually added that to the member method but you are right, it makes more sense here, I'm going to change it